openvoice_app.py

```diff
@@ -14,7 +14,11 @@
 en_ckpt_base = 'checkpoints/base_speakers/EN'
 zh_ckpt_base = 'checkpoints/base_speakers/ZH'
 ckpt_converter = 'checkpoints/converter'
-device = 'cuda' if torch.cuda.is_available() else 'cpu'
+device = 'cpu'
+if torch.cuda.is_available():
+    device = 'cuda'
+if torch.backends.mps.is_available():
```

Comment:
Hi @fakerybakery, Thanks for the PR. Could you please verify if this setting `device = 'mps'` is really effective? Additionally, it would be great if you could confirm its validity.

Reply:
Hi, `device='mps'` makes the program utilize the Apple Silicon GPU

Reply:
Hello @fakerybakery , We've encountered some issues with dimensionality when using **mps**. You can find a in-depth discussion on this topic [here](https://github.com/iamshreeram/OpenVoice/discussions/1#discussioncomment-8004304).

Could you please make sure that the code runs smoothly and that the **Apple GPU** is being properly utilized while working with **mps**?

We kindly request you to try using **mps** as the device type in this [notebook](https://github.com/myshell-ai/OpenVoice/blob/main/demo_part1.ipynb) and let us know if it functions correctly.

We greatly appreciate your effort.

Reply:
Sorry, there is probably a compatibility issue, i'll close this